src/modify_resources.py

```diff
@@ -34,23 +34,27 @@
 '''Stressing the Network'''
 # Container_to_bandwidth maps Docker container id to the bandwidth that container should be throttled to.
 # Assumes that the container specified by container id is located in the machine for ssh_client
-# Bandwidth units: Mbps
+# Bandwidth units: kbps
 def set_egress_network_bandwidth(ssh_client, container_id, bandwidth):
     interface_name = get_container_veth(ssh_client, container_id)
-    
+
     # Execute the command within OVS
     # OVS policy bandwidth accepts inputs in kbps
-    bandwidth_kbps = bandwidth * (10 ** 3)
-    ovs_policy_cmd = 'ovs-vsctl set interface {} ingress_policing_rate={}'.format(interface_name, bandwidth_kbps)
+    bandwidth_kbps = bandwidth[container_id]
```

Comment:
For the sake of simplicity, let us change the arguments for the units of bandwidth throttling to bytes/sec. Then you should have some sort of general conversion within the modify_resources class that changes bytes to whatever unit is used to modify the resources. This will simplify the API considerably.